packages/nc-gui-v2/pages/index/index.vue

```diff
@@ -61,97 +61,113 @@ onMounted(() => {
-          :placeholder="$t('activity.searchProject')"
-        ></a-input-search>
-        <div class="flex-grow"></div>
-
-        <a-dropdown v-if="isUIAllowed('projectCreate', true)" @click.stop>
-          <a-button class="nc-new-project-menu !shadow">
-            <div class="flex align-center">
-              {{ $t('title.newProj') }}
-              <MdiMenuDown class="menu-icon" />
-            </div>
-          </a-button>
-
-          <template #overlay>
-            <a-menu>
-              <div
-                v-t="['c:project:create:xcdb']"
-                class="grid grid-cols-12 cursor-pointer hover:bg-gray-200 flex items-center p-2 nc-create-xc-db-project"
-                @click="navigateTo('/project/create')"
-              >
-                <MdiPlus class="col-span-2 mr-1 mt-[1px] text-primary text-lg" />
-                <div class="col-span-10 text-sm xl:text-md">{{ $t('activity.createProject') }}</div>
-              </div>
-              <div
-                v-t="['c:project:create:extdb']"
-                class="grid grid-cols-12 cursor-pointer hover:bg-gray-200 flex items-center p-2 nc-create-external-db-project"
-                @click="navigateTo('/project/create-external')"
-              >
-                <MdiDatabaseOutline class="col-span-2 mr-1 mt-[1px] text-green-500 text-lg" />
-                <div class="col-span-10 text-sm xl:text-md" v-html="$t('activity.createProjectExtended.extDB')" />
-              </div>
-            </a-menu>
-          </template>
-        </a-dropdown>
-      </div>
-
-      <div v-if="isLoading">
-        <a-skeleton />
-      </div>
-
-      <a-table
-        v-else
-        :custom-row="
-          (record) => ({
-            onClick: () => {
-              $e('a:project:open')
-              navigateTo(`/nc/${record.id}`)
-            },
-          })
-        "
-        :data-source="filteredProjects"
-        :pagination="{ position: ['bottomCenter'] }"
-      >
-        <!-- Title -->
-        <a-table-column key="title" :title="$t('general.title')" data-index="title">
-          <template #default="{ text }">
-            <div class="capitalize !w-[400px] overflow-hidden overflow-ellipsis whitespace-nowrap nc-project-row" :title="text">
-              {{ text }}
-            </div>
-          </template>
-        </a-table-column>
-        <!-- Actions -->
-        <a-table-column key="id" :title="$t('labels.actions')" data-index="id">
-          <template #default="{ text, record }">
-            <div class="flex align-center">
-              <MdiEditOutline
-                v-t="['c:project:edit:rename']"
-                class="nc-action-btn"
-                @click.stop="navigateTo(`/project/${text}`)"
-              />
-              <MdiDeleteOutline class="nc-action-btn" @click.stop="deleteProject(record)" />
-            </div>
-          </template>
-        </a-table-column>
-      </a-table>
-    </a-card>
+    <a-row>
+      <a-col :span="8"></a-col>
```

Comment:
a full empty column? Is this intended?

Reply:
Just copying the layout from v1. Each column contains 8 spans. I believe there is a better way to do it..

[ empty ] [ my project ] [ sponsors + social ]